### PR TITLE
[JW8-10171] Add localization support for Keyboard Shortcuts

### PR DESCRIPTION
**Pull request by DanFerrer**
### This PR will...

- Add  int/localization support for Keyboard shortcuts
- Support Greek (el) language

### Why is this Pull Request needed?

More localization support

### Are there any points in the code the reviewer needs to double check?

How do we want the shortcuts menu to look when its customized/translated and a menu item has longer text? I currently have it set to so the viewer has to scroll horizontally to view text. Should we make the menu bigger instead @aliciahurst 

### Are there any Pull Requests open in other repos which need to be merged with this?

https://github.com/jwplayer/jwplayer-commercial/pull/6988

#### Addresses Issue(s):

JW8-10171

### CODE REVIEW

**Comment by johnBartos** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  🗿 Set a milestone. It should be the ticket's fix version in JIRA.
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: src/css/controls/imports/modal.less [hunk 1]
```diff
@@ -20,6 +20,6 @@
         width: 100%;
         max-height: none;
         max-width: none;
-        z-index: 1;
+        z-index: 2;
```
Comment (jnatalzia) [inline src/css/controls/imports/modal.less]: needed?

File: src/css/controls/imports/shortcuts-tooltip.less [hunk 2]
```diff
@@ -51,32 +51,30 @@
     .jw-shortcuts-tooltip-list {
         display: flex;
         max-width: 340px;
+        font-size: 12px;
+        margin-left: 10px;
```
Comment (jnatalzia) [inline src/css/controls/imports/shortcuts-tooltip.less]: `margin: auto 10px`?

File: src/css/controls/imports/modal.less (hunk 1 above)
Comment (DanFerrer) [reply src/css/controls/imports/modal.less]: The time slider is shown on top of the shortcuts menu when its open on smaller players. Increasing the Z-index by 1 fixes it.

File: src/css/controls/imports/shortcuts-tooltip.less [hunk 3]
```diff
@@ -51,32 +51,29 @@
     .jw-shortcuts-tooltip-list {
         display: flex;
         max-width: 340px;
+        font-size: 12px;
+        margin: 0, 10px;
```
Comment (jnatalzia) [inline src/css/controls/imports/shortcuts-tooltip.less]: shouldn't be a comma here

**Pull request merged**